PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -452,6 +453,10 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
         if postfix != "" and metType == "PF" and not hasattr(process, 'pat'+metType+'Met'+postfix):
             noClonesTmp = [ "particleFlowDisplacedVertex", "pfCandidateToVertexAssociation" ]
             configtools.cloneProcessingSnippet(process, getattr(process,"producePatPFMETCorrections"), postfix, noClones = noClonesTmp, addToTask = True)
+            #MM FIXME, this could be done in a mch better way, this is not
+            #handled by the above cloning sequence
+            getattr(process,"selectedPatJetsForMetT1T2Corr").src=jetCollectionUnskimmed #cms.InputTag("patJets"+postfix)
```

Comment:
the result of this call does not quite make sense to me. In the standard config I see the following:
- selectedPatJetsForMetT1T2Corr has src = slimmedJetsPuppi
- selectedPatJetsForMetT1T2CorrPuppi has patJetsNoHF
- selectedPatJetsForMetT1T2CorrNoHF has (unchanged) patJets

All of the above had patJets in the baseline setup.

Reply:
these modules are not running in the default (mini)AOD configuration.
Is there something else missing in the configuration that would enable them or are they instantiated for no particular reason?